fastify.d.ts

```diff
@@ -529,6 +547,12 @@ declare namespace fastify {
      */
     addHook(name: 'preValidation', hook: FastifyMiddleware<HttpServer, HttpRequest, HttpResponse>): FastifyInstance<HttpServer, HttpRequest, HttpResponse>
 
+    /**
+     * Hook that is fired after a request is processed, but before the response is serialized
+     * hook
+     */
+    addHook(name: 'preSerialization', hook: (this: FastifyInstance<HttpServer, HttpRequest, HttpResponse>, req: FastifyRequest<HttpRequest>, reply: FastifyReply<HttpResponse>, payload: any, done: (err?: Error, value?: any) => void) => void): FastifyInstance<HttpServer, HttpRequest, HttpResponse>
```

Comment:
I think this should be the same type that is passed to the preSerialization option.

Reply:
Good point. Changed.

Reply:
I cannot see any changes here.

Reply:
Ehm... I mean "I have it changed it locally and it will be there in my next push". Sorry for the confusion.

Reply:
No the commits are there.